lib/plugins/aws/package/compile/events/stream.js

```diff
@@ -69,6 +69,12 @@ class AwsCompileStreamEvents {
               required: ['onFailure'],
             },
             tumblingWindowInSeconds: { type: 'integer', minimum: 0, maximum: 900 },
+            patterns: {
+              type: 'array',
+              minItems: 1,
+              maxItems: 5,
```

Comment:
I think is should be increased to 10 as per official docs and ability to increase the quota to 10